apps/examples/artik_demo/artik_demo.c

```diff
@@ -747,6 +747,7 @@ int send_data_to_artik(int *data)
 		if (ret < 0) {
 			/* ERROR: Connection Failed */
 			ndbg("ERROR: connect failed: %d\n", ret);
+			close(sockfd);
```

Comment:
Below 3 lines are duplicated.
close(sockfd);
free(client_tls);
return ERROR;

You had better use 'goto error_with_fd' and do this step there.

Reply:
Thanks. Will update it.